src/main/java/core/selenium/config/PropertiesReader.java

```diff
@@ -0,0 +1,54 @@
+package core.selenium.config;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * PropertiesReader class.
+ */
+public class PropertiesReader {
+    private Properties property;
+    private FileReader reader;
+
+    /**
+     * Initializes an instance of properties files.
+     *
+     * @param propertiesPath
+     */
+    public PropertiesReader(final String propertiesPath) {
+        try {
+            reader = new FileReader(propertiesPath);
+            property = new Properties();
+            property.load(reader);
+        } catch (FileNotFoundException e) {
+            System.out.println(e.getMessage());
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        } finally {
+            closeReader();
+        }
+    }
+
+    /**
+     * Gets environment property.
+     * @param env
+     * @return property value.
+     */
+    protected String getEnvProperty(final String env) {
```

Comment:
I don't think this method should be in this classs, will all the values be a system property?